discord/invite.go

```diff
@@ -32,6 +32,16 @@ type Invite struct {
 	InviteMetadata
 }
 
+// URL returns a Discord invite URL linking to the invite.
+func (i Invite) URL() string {
+	return "https://discord.gg/" + i.Code
```

Comment:
Can we have a constant for this URL?

Reply:
Yeah I didn't think about that. Which package do you think it should be defined in? 

Should we also use constants for `cdn.discordapp` URLs too? e.g. [`emoji.go`](https://github.com/diamondburned/arikawa/blob/v3/discord/emoji.go#L87) 
There's actually quite a few instances of plain-text `cdn.discordapp` URLs scattered around.

Reply:
invite.go is good.

> Should we also use constants for `cdn.discordapp` URLs too? e.g. [`emoji.go`](https://github.com/diamondburned/arikawa/blob/v3/discord/emoji.go#L87)
> There's actually quite a few instances of plain-text `cdn.discordapp` URLs scattered around.

That'd be great, but it can be a separate PR.